### DISCUSSION TITLE
Making guinavbar/guitabbar an overlay when set to "onupdate"

### DISCUSSION THREAD

**Discussion by aleksmark98** [Jelmerro/Vieb#531] [Q&A] [answered]
I prefer the least amout of GUI clutter possible when using an app and so I have the guinavbar and guitabbar set to "onupdate". This option is great and is like 30% of why I love Vieb.

Unfortunatelly, whenever I switch tabs or use the explore mode, the entire page jumps down and then jumps up as these gui features hide. This feels quite disruptive.
I think it would be a lot smoother if the guitabbar and guinavbar poped up as an overlay over the page instead of moving the page downwards.

**Comment by Jelmerro** [comment]
This would be a nice addition, but I'm undecided if this would be better as a setting or done via colorschemes. I'll think about this more in the coming weeks.

**Reply by ghost** [reply, marked as answer]
Any updates on this? I use this simple stylesheet as a temporal solution plus `set guinavbar=oninput`.

```css
#navbar {
    position: absolute;
}

#suggest-dropdown {
    top: 0;
}
```

It overlaps the tabbar when in use, which i don't find to be a big deal.

**Reply by Jelmerro** [reply]
I did not decide on which I think is better yet, but thank you for commenting a solution that works. I don't think this will be added to the base config, but it is good to have a working snippet here ready to use.
